templates/index.html

```diff
@@ -165,12 +165,11 @@ <h3 class="p-heading-icon__title">Vanilla Framework</h3>
     <div class="p-card--highlighted col-4 col-medium-3">
       <div class="p-heading-icon--small">
         <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/9a5b9c51-sketch-icon.svg" alt="Sketch library:" />
-          <h3 class="p-heading-icon__title">Vanilla Design</h3>
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/c42ba977-figma-logo-o.svg" alt="Figma library:" />
+          <h3 class="p-heading-icon__title">Vanilla UI kit</h3>
         </div>
-        <p>Get our Sketch library of common design components.</p>
-        <p style="margin-bottom: 0px;"><a onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage', 'eventAction' : 'Downloads', 'eventLabel' : 'Download Vanilla Design UI Kit', 'eventValue' : undefined });" class="p-button" href="https://assets.ubuntu.com/latest-redirects/vanilla-framework.sketch">Download the Sketch UI Kit</a></p>
-        <small><i class="p-icon--warning"></i> The Sketch UI Kit is no longer maintained. Current version is compatible with Vanilla v2.25.</small>
+        <p>Get our UI kit to start designing with Vanilla components.</p>
+        <p style="margin-bottom: 0px;"><a onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage', 'eventAction' : 'Downloads', 'eventLabel' : 'Download Vanilla Design UI Kit', 'eventValue' : undefined });" class="p-button" href="https://www.figma.com/community/file/1435297834108003391/vanilla-core-component-library">Download the Figma UI kit</a></p>
```

Comment:
@dgtlntv Should the button say "Download"? Can it be "downloaded" or is it used online with Figma? Maybe "Get the Figma UI kit" would work better?

Reply:
Hmm yeah good point. I was thinking of staying consistent with the github button. But you are right, that for figma it doesnt make a lot of sense since everything is in the cloud ^^ I'll change it :)